html/canvas/offscreen/manual/draw-generic-family/2d.text.draw.generic.family.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<title>OffscreenCanvas test: 2d.text.draw.generic.family</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/html/canvas/resources/canvas-tests.js"></script>
+<script>
+function drawCanvas(ctx, family)
+{
+    ctx.font = '32px ' + family;
+    ctx.fillText(family, 0, 32);
+}
+
+function testDrawGenericFamily(family)
+{
+    var offscreenCanvas = new OffscreenCanvas(300, 150);
+    var oCtx = offscreenCanvas.getContext('2d');
+    drawCanvas(oCtx, family);
+    var canvas = document.createElement('canvas');
+    var ctx = canvas.getContext('2d');
+    drawCanvas(ctx, family);
+
+    var data1 = oCtx.getImageData(0, 0, 300, 150).data;
+    var data2 = ctx.getImageData(0, 0, 300, 150).data;
+    assert_array_equals(data1, data2,
+      "The image data generated by drawing generic font family '" + family +
+      "' should be the same for both OffscreenCanvas and regular canvas");
+}
+
+test(function() {
+    testDrawGenericFamily('sans-serif');
+    testDrawGenericFamily('serif');
+    testDrawGenericFamily('cursive');
+    testDrawGenericFamily('fantasy');
+    testDrawGenericFamily('monospace');
+    testDrawGenericFamily('cursive');
+}, "Test that drawing generic font families produces the same result between canvas and OffscreenCanvas");
```

Comment:
Please reorganize this a tiny bit so it creates one test per family rather than testing them all in a single test. Same comment for the worker test.

Reply:
Sure thing.